test/test_multiprocessing.py

```diff
@@ -833,12 +833,21 @@ def test_cuda_parameter_sharing(self):
     @unittest.skipIf(NO_MULTIPROCESSING_SPAWN, "Disabled for environments that \
                      don't support multiprocessing with spawn start method")
     def test_integer_parameter_serialization(self):
-        iparam = torch.nn.Parameter(torch.tensor(0, dtype=torch.int64), requires_grad=False)
+        for device in ['cpu', 'cuda']:
```

Comment:
Test errors are real, you can skip this test if cuda ipc is not available, like the tests above. Also, nit, prefer using tuples ('cpu', 'cuda') over lists.

Reply:
@ngimel oops! my mistake. Thanks for the advice.

I split the test in two: for CPU and for CUDA. The first one is going to be run regardless of the availability of CUDA IPC. The second test is skipped if CUDA is not available.